api/v1alpha1/clienttrafficpolicy_types.go

```diff
@@ -59,6 +59,15 @@ type ClientTrafficPolicySpec struct {
 	//
 	// +optional
 	EnableProxyProtocol *bool `json:"enableProxyProtocol,omitempty"`
+	// HTTP3Settings provides HTTP/3 configuration on the listener.
+	// +optional
+	HTTP3Settings *HTTP3Settings `json:"http3Settings,omitempty"`
+}
+
+type HTTP3Settings struct {
+	// Enabled enables HTTP/3 support on the listener.
+	// Disabled by default.
+	Enabled bool `json:"enabled,omitempty"`
```

Comment:
`Enabled` is not needed, instantiating the stunt should implicitly enable `http3`